i/ipywidgets/build_info.json

```diff
@@ -2,17 +2,20 @@
     "maintainer":"Vinod.K1@ibm.com",
     "package_name" : "ipywidgets", 
     "github_url": "https://github.com/jupyter-widgets/ipywidgets",
-    "version": "8.1.1",
+    "version": "8.1.2",
     "default_branch": "main",
     "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
     "package_dir": "i/ipywidgets",
-    "build_script": "ipywidgets_ubi_8.7.sh",
+    "build_script": "ipywidgets_ubi_9.3.sh",
     "validate_build_script": "true",
     "use_non_root_user": "false",
     "docker_build": "true",
-
     "8.*.*" : {
```

Comment:
@vinodk99 check again pattern you have. 
for 8.1.2 both will work and and 8.*.* will take precendenc

Reply:
@nikhil-kalbande, Updated the build_info.json as suggested.